README.md

```diff
@@ -64,6 +64,7 @@ $ go get github.com/apache/incubator-openwhisk-wskdeploy  # see known issues bel
 Use the Go utility to build the ```wskdeploy``` binary as follows:
 ```sh
 $ cd src/github.com/apache/incubator-openwhisk-wskdeploy/
+$ go get
```

Comment:
Please replace `go get` with `godep get` since go get pulls down latest version of all the dependencies instead of honoring the version set in deps JSON file.